hunting-buddy.lic

```diff
@@ -259,6 +264,17 @@ class HuntingBuddy
         echo('***STATUS*** stopping due to time')
         break
       end
+      if Flags['familiar_drag'] && @bail_on_fam
```

Comment:
❓ If I'm reading this right, this code will always change behavior. Do you think any WM may want to do neither of these things?

Reply:
When the familiar drags you, you end up in another hunting room. You could end up in a room with someone else not on your hunting buddies list. I spoke with @dr-leustyin  and @BinuDR. @dr-leustyin  suggested ending the hunt and @BinuDR  suggest returning to the room drug from. My goal is to return to the room but ending the hunt is an option because you are most-likely bashed up enough to keep getting stuns and then more serious injuries quickly.